roadmap/implementers-guide/src/node/backing/candidate-backing.md

```diff
@@ -27,8 +27,6 @@ The [Collator Protocol][CP] subsystem is the primary source of non-overseer mess
 
 This subsystem requests validation from the [Candidate Validation][CV] and generates an appropriate [`Statement`][Statement]. All `Statement`s are then passed on to the [Statement Distribution][SD] subsystem to be gossiped to peers. When [Candidate Validation][CV] decides that a candidate is invalid, and it was recommended to us to second by our own [Collator Protocol][CP] subsystem, a message is sent to the [Collator Protocol][CP] subsystem with the candidate's hash so that the collator which recommended it can be penalized.
 
-The subsystem should maintain a set of handles to Candidate Backing Jobs that are currently live, as well as the relay-parent to which they correspond.
```

Comment:
candidate backing still uses the job terminology in the code, so it's fine to keep it

Reply:
Yes, I was actually confused about this. Thanks for explaining!